ansible/ansible.cfg

```diff
@@ -34,7 +34,7 @@ display_skipped_hosts	= false
 
 
 # Default strategy
-strategy		= free
+strategy		= linear
```

Comment:
Maybe add a comment why each task should be blocking, and `free` strategy is not applicable?

Reply:
because the role is considered as plug and play there are a lot of varibels are defined in runtime and host can fail if variable not yet defined. 
`linear` - is default strategy, the `ansible.cfg` was coped from different ansible there `free` was set to speed up the process.